projects/plugins/boost/app/assets/src/js/pages/settings/elements/CriticalCssShowStopperError.svelte

```diff
@@ -72,11 +74,7 @@
 
 	<div slot="actionButton">
 		{#if $criticalCssStatus.retried_show_stopper}
-			<a
-				class="button button-secondary"
-				href="https://wordpress.org/support/plugin/jetpack-boost/"
-				target="_blank"
-			>
+			<a class="button button-secondary" href={supportLink} target="_blank">
```

Comment:
Do we normally validate that Svelte Props have been provided? Might be an argument to move to TypeScript for Svelte files, which allows us to make props a required value.

Reply:
Good catch - yeah, we might not need to validate it after all if Svelte doesn't compile otherwise. Didn't think of that.


Well then - dismiss this bit 🙂

Reply:
Good points, both.
In fact, Svelte compilation succeeds but shows an error at runtime. However, adding a default value to the support link makes the validation unnecessary and always ensures that the supportLink exists. Updated this now.